client/main.lua

```diff
@@ -191,6 +192,14 @@ local function startTargeting()
                         end
                     end
 
+                    if option.menuName == nil then
+                        option.menuName = 'home'
```

Comment:
Shouldn't need to set `'home'` at all. Remove other parts setting it as well and refer to other review comment below.